sdks/java/build-tools/src/main/resources/beam/suppressions.xml

```diff
@@ -88,6 +88,10 @@
   <suppress id="ForbidNonVendoredGuava" files=".*zetasql.*ExpressionConverter\.java" />
   <suppress id="ForbidNonVendoredGuava" files=".*zetasql.*ZetaSQLPlannerImpl\.java" />
   <suppress id="ForbidNonVendoredGuava" files=".*zetasql.*SqlAnalyzer\.java" />
+  <suppress id="ForbidNonVendoredGuava" files=".*pubsublite.*AddUuidsTransform\.java" />
+  <suppress id="ForbidNonVendoredGuava" files=".*pubsublite.*MessageCoderTest\.java" />
+  <suppress id="ForbidNonVendoredGuava" files=".*pubsublite.*PubsubLiteSink\.java" />
+  <suppress id="ForbidNonVendoredGuava" files=".*pubsublite.*UuidDeduplicationTransformTest\.java" />
```

Comment:
They should not be required since they seem to be using the import as an implementation detail within the class/method. They should instead depend on the vendored version by importing `org.apache.beam.vendor.guava.v26_0_jre....`

Reply:
This is not accurate. The client library uses guava ImmutableListMultimap as an input type to represent its attributes: these files need to construct an ImmutableListMultimap and thus need guava.

Reply:
I see. I would highly suggest to use APIs that don't depend on guava or for the client library to fix its API as guava is a constant source of compatibility issues for users.

Reply:
I don't think its reasonable to avoid guava in the client library in this situation. Doing so leaves users of the client library with a worse api (Map<String, List<ByteString>> instead of ListMultimap<String, ByteString>) for no gain. There is clearly already a non-vendored version of guava within beam, and this library uses the one is used as a common dependency for all google cloud client libraries (i.e. there should always be a version usable by it and all other cloud client libraries). I don't think there's anything to be gained here by removing guava in the public api.

Reply:
I believe this is resolved- the api surface now operates on the proto messages themselves due to the added complexity to users WRT dependency issues being more important than the awkwardness of representing a multimap as a list of lists.